scan.py

```diff
@@ -56,18 +56,22 @@ def parse_line(rowData, formatToActorMap):
 
     timeAndSyllablesIter = iter(timeAndSyllables)
     for i, (val1, val2) in enumerate(zip(timeAndSyllablesIter, timeAndSyllablesIter)):
-        syllables.append({
+        syllable = {
             'len': int(val1['formattedValue']),
             'text': val2['formattedValue']
-        })
-
+        }
+        
         currActor = formatToActorMap[color_to_hex(val2['effectiveFormat']['backgroundColor'])]
         if not actors or currActor != actors[-1]:
             actors.append(currActor)
             breakpoints.append(i)
-
-    return {
+            syllable['ifx'] = currActor
+        
+        syllables.append(syllable)
+    
+    parsed_line = {
         'en': values[get_column_idx('B')]['formattedValue'],
+        'karaoke': values[get_column_idx('D')].get('formattedValue'),
```

Comment:
This seems unnecessary given lines 84-87?

Reply:
On second thought, I think 84-85 are actually the unnecessary lines, oops!